yt/visualization/plot_container.py

```diff
@@ -139,8 +139,12 @@ def get_symlog_minorticks(linthresh, vmin, vmax):
         the maximum value in the colorbar
 
     """
-    if vmin >= 0 or vmax <= 0:
+    if vmin > 0 or vmax < 0:
         return get_log_minorticks(vmin, vmax)
+    elif vmin == 0:
+        return np.hstack( (0, get_log_minorticks(linthresh, vmax)) )
+    elif vmax == 0:
+        return np.hstack( (-get_log_minorticks(linthresh,-vim)[::-1], 0) )
```

Comment:
There's a typo here (that the tests are catching and complaining about) `vim` is a typo, I think you meant `vmin`